src/OrchardCore.Modules/OrchardCore.Security/SecurityHeaderDefaults.cs

```diff
@@ -0,0 +1,57 @@
+using OrchardCore.Security.Options;
+
+namespace OrchardCore.Security
+{
+    public static class SecurityHeaderDefaults
+    {
+        public static readonly string[] ContentSecurityPolicy = new[]
+        {
+            $"{ContentSecurityPolicyValue.BaseUri} {ContentSecurityPolicyOriginValue.None}",
```

Comment:
We need to check these default locally and possible live to avoid conflict with `OC.Resources` CDNs

Reply:
As I said, after adding the header the styles, scripts are blocked, I tried to change the origin to `any` everything works fine except `all.css` for unknown reason

For that we need to not inject the headers unless the admin works fine or let the user enable header values from UI

Reply:
CSP is more complicated in terms of options I may suggest to add textbox beside each option, when its empty the option will be ignored